pretix_sepadebit/models.py

```diff
@@ -15,3 +16,11 @@ class SepaExportOrder(models.Model):
     order = models.ForeignKey('pretixbase.Order', on_delete=models.CASCADE)
     payment = models.ForeignKey('pretixbase.OrderPayment', on_delete=models.CASCADE, null=True)
     amount = models.DecimalField(max_digits=10, decimal_places=2)
+
+
+class SepaDueDate(models.Model):
+    payment = models.OneToOneField('pretixbase.OrderPayment', on_delete=models.CASCADE, null=True, related_name='due')
```

Comment:
I think ``sepadebit_due`` would be a better related name to avoid clashes with other plugins